autode/geom.py

```diff
@@ -315,6 +315,11 @@ def symm_matrix_from_ltril(array):
     Returns:
         (np.ndarray):
     """
+
+    if len(array) > 0 and type(array[0]) in (list, np.ndarray):
+        # Flatten the array of arrays
+        array = [item for sublist in array for item in sublist]
```

Comment:
is np.flatten helpful here (or array.flatten, whatever it is)

Reply:
sadly not, as the array could be jagged, making numpy sad:

```
>>> np.array([[1], [1, 2]]).flatten()
array([list([1]), list([1, 2])], dtype=object)
```